types.go

```diff
@@ -19,8 +19,8 @@ type Prefix []byte
 // hash from the one used for interior nodes.
 type KeyValuePair struct {
 	_struct bool        `codec:",toarray"`
-	Key     Hash        `codec:"key"`
-	Value   interface{} `codec:"value"`
+	Key     Hash        `codec:"k"`
+	Value   interface{} `codec:"v"`
```

Comment:
Sure thing will make that change, good idea.

On Wednesday, February 3, 2016, Andy Alness <notifications@github.com>
wrote:

> In types.go
> <https://github.com/keybase/go-merkle-tree/pull/1#discussion_r51758629>:
>
> > @@ -19,8 +19,8 @@ type Prefix []byte
> >  // hash from the one used for interior nodes.
> >  type KeyValuePair struct {
> >  	_struct bool        `codec:",toarray"`
> > -	Key     Hash        `codec:"key"`
> > -	Value   interface{} `codec:"value"`
> > +	Key     Hash        `codec:"k"`
> > +	Value   interface{} `codec:"v"`
>
> I'm wondering if maybe making the Value a byte array is simpler? I don't
> have it written this way right now but I was thinking of converting my
> values to byte arrays-- for public nodes I have a clearly defined struct
> but for private nodes I just have a blob of encrypted data. I also already
> know the types because the folder IDs (my Keys) contain that information.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/go-merkle-tree/pull/1/files#r51758629>.
>